app/models/communication.rb

```diff
@@ -60,6 +61,20 @@ def self.create_with_twilio_details!(appointment:, twilio_sid:, twilio_msg_statu
     end
   end
 
+  def self.create_with_imo_details!(appointment:, notification:, result:)
+    transaction do
```

Comment:
Looks like the transaction in the Twilio method was because we were creating two records. Here we're only making one, so we can remove the transaction.

PS. Do we need an equivalent of `TwilioSmsDeliveryDetail`? I'm not saying we do, I'm genuinely unaware. What's that thing for?

Reply:
There will be a `TwilioSmsDeliveryDetail`. That's why it's still wrapped in a transaction. It's in an upcoming ticket.